src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -74,7 +74,12 @@ public ObservableList<Person> getFilteredPersonList() {
 
     @Override
     public Path getAddressBookFilePath() {
-        return model.getAddressBookFilePath();
+        return model.getUniCashFilePath();
```

Comment:
I believe this part is still kept to support the other parts of the codebase using this right? Can be removed when we remove AddressBook related stuff in the future?

Reply:
Is it possible to just shift all references to the new getUniCashFilePath in this iteration or would that be too large of a mgiration

Reply:
It's possible to delete it actually, I just did backwards support in case but I believe there's no more usages of the `getAddressBookFilePath` method